### PR TITLE
Add pii orb

### PR DESCRIPTION
**Pull request by HerveMorosini**
Create an orb to execute PII tests on each microservice.

**When using the pii orb, you have to pass following parameters:**
- service (service name)
- channel_id (Slack channel id retrieve from Copy link on srv-<service name>-dev slack channel)

**PII tests executed on service:**
- Execute tests on List of PII descriptions on service
- Execute tests on Get User PII on existing user on service
- Create user AND Execute tests on Get User PII on created user on service
- Delete user AND Execute tests on Get User PII on deleted user on service

### CODE REVIEW

File: orbs/pii/orb.yml [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+          FORCE_COLOR: "0"
+
+commands:
+  clone_e2e_jt_tests:
+    description: "Clone e2e_jt_tests repository"
+    steps:
+      - run:
+          name: "Clone e2e_jt_tests repository"
+          command: |
+            git clone -b master git@github.com:jobteaser/e2e_jt_tests.git
```
Comment (happypoulp) [inline orbs/pii/orb.yml:68]: **Reminder: ⚠️  All code in the present repo is public.**

Please remove this and make the repository URL a parameter.

File: orbs/pii/orb.yml [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+          working_directory: "~/project"
+
+  clone_privacy:
+    description: "Clone privacy repository"
+    steps:
+      - run:
+          name: "Clone privacy repository"
+          command: |
+            git clone -b master git@github.com:jobteaser/privacy.git
```
Comment (happypoulp) [inline orbs/pii/orb.yml:77]: **Reminder: ⚠️  All code in the present repo is public.**

Please remove this and make the repository URL a parameter.

**Review by nicosvirjt — APPROVED** [review]
🥳

**Review by ejoubaud — APPROVED** [review]
This is looking great! 👏 Impressive how clear you made this complex process 🤩 I second @happypoulp 's remark about the repo, but otherwise 👍

File: orbs/pii/orb.yml (hunk 1 above)
Comment (HerveMorosini) [reply orbs/pii/orb.yml:68]: Info from Yohan:
`Hello Hervé, après quelques discussions, on va de notre côté essayer de passer le repo circleci en privé, mais ça va prendre un peu de temps. En attendant, on a donné le feu vert pour merger la PR en l'état
`

File: orbs/pii/orb.yml (hunk 2 above)
Comment (HerveMorosini) [reply orbs/pii/orb.yml:77]: Info from Yohan:
`Hello Hervé, après quelques discussions, on va de notre côté essayer de passer le repo circleci en privé, mais ça va prendre un peu de temps. En attendant, on a donné le feu vert pour merger la PR en l'état
`

**Pull request merged**